django/bosscore/views/views_resource.py

```diff
@@ -28,13 +28,16 @@
 from bosscore.lookup import LookUpKey
 from bosscore.permissions import BossPermissionManager
 from bosscore.privileges import check_role
+from bosscore.public_channels import PUBLIC_CHANNELS
 
 from bosscore.serializers import CollectionSerializer, ExperimentSerializer, ChannelSerializer, \
     CoordinateFrameSerializer, CoordinateFrameUpdateSerializer, ExperimentReadSerializer, ChannelReadSerializer, \
     ExperimentUpdateSerializer, ChannelUpdateSerializer, CoordinateFrameDeleteSerializer
 
 from bosscore.models import Collection, Experiment, Channel, CoordinateFrame, Source
 
+public_channel_frozen_set
```

Comment:
Looking at this on my phone but this looks like a mistake?

Reply:
Nice catch, which was my first idea and I decided to go with a separate file. I removed it.